client/src/main/scala/eventstore/CoreCompat.scala

```diff
@@ -0,0 +1,190 @@
+package eventstore
+
+import eventstore.core.{settings ⇒ cs}
+import eventstore.{core ⇒ c}
+
+trait CoreCompat {
+
+  private lazy val RM: Boolean  = Settings.Default.requireMaster
```

Comment:
IMHO these `RM , RLT, RBS` - are confusing names.

Reply:
I'll fix that

Reply:
@t3hnar Fixed